.github/workflows/slow.yaml

```diff
@@ -181,3 +181,59 @@ jobs:
         with:
           name: build-logs-freebsd-${{ matrix.osversion }}
           path: btlayer-*.log
+
+  openbsd:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v4
+
+      - name: Run test-builds
+        id: test-builds
+        uses: vmactions/openbsd-vm@v1
+        with:
+          usesh: true
+          prepare: |
+            pkg_add \
+              autoconf-2.72p0 \
+              autoconf-archive \
+              automake-1.16.5 \
+              bash \
+              coreutils \
+              cppunit \
+              git \
+              ggrep \
+              gmake \
+              libltdl \
+              libtool \
+              libnettle \
+              libxml \
+              libtool \
+              m4 \
+              metaauto
+
+            mkdir $HOME/bin
+            ln -s /usr/local/bin/ggrep $HOME/bin/grep
+
+          run: |
+            export MAKE=gmake
+            export pjobs="-j`gnproc`"
+            export AUTOMAKE_VERSION=1.16
+            export amver=${AUTOMAKE_VERSION}
+            export ACLOCAL_AUTOMAKE_DIR="/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
```

Comment:
Is there a similar hack that can avoid the following [warnings](https://github.com/kinkie/squid/actions/runs/12354857472/job/34477146806)?

```
  aclocal-1.16: warning: couldn't open directory '/usr/bin/../share/aclocal': No such file or directory
```

Reply:
Not that I know of. OpenBSD is rough around the edges in these areas